### PR TITLE
NMS-15757: Allow metadata in Xml collector

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://opennms.atlassian.net/browse/NMS-15757

### CODE REVIEW

File: docs/modules/reference/pages/performance-data-collection/collectors/xml.adoc [hunk 1]
```diff
@@ -76,7 +76,10 @@ XML collections are defined in `$\{OPENNMS_HOME}/etc/xml-datacollection-config.x
 </xml-collection>
 ----
 
-The `url` attribute of the `<xml-source>` supports node and IpInterface <<reference:configuration/filters/parameters.adoc#filter-parameters, field placeholders>>, though it does not support metadata tokens.
+The `url` attribute of the `<xml-source>` supports node and IpInterface <<reference:configuration/filters/parameters.adoc#filter-parameters, field placeholders>>.
+
+NOTE: Metadata expressions can also be used in attributes of the `xml-datacollection-config.xml` configuration file.
+This allows the user to also reference credentials stored in the secure credentials
```
Comment (Bonrob2) [inline docs/modules/reference/pages/performance-data-collection/collectors/xml.adoc:82]:
```suggestion
This lets the user to also reference credentials stored in the secure credentials.
```
Allows is perfectly correct, but NH style guide suggests "lets" instead, so I've been trying to update it.

**Review by Bonrob2 — COMMENTED** [review]
one minor (silly) change in the docs. Thanks!

**Review by Bonrob2 — APPROVED** [review]
doc update LGTM. Thanks!

**Review by RangerRick — APPROVED** [review]
`Interpolator.pleaseInterpolate()` -- so polite

**Pull request merged**